examples/marklogic-cloud-connection.js

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2023 MarkLogic Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* This file provides an example for the users to help them connect to MarkLogic cloud using the apiKey and host
+and writes a document in database - Documents */
+
+const marklogic = require('../');
+
+let db = marklogic.createDatabaseClient({
+    // All the below key-value pairs are required
+    apiKey:     'apiKey',
+    host:     'example.marklogic.com',
+    authType: 'cloud',
+    basePath: 'basePath'
+});
+
+let writeObject = {
+    uri: '/write/string1.json',
+    contentType: 'application/json',
```

Comment:
This looks good, but just curious - is `contentType` required here? My guess would have been that MarkLogic assumes a mimetype based on the URI when no content type is given.

Reply:
"contentType" is not required. Verified it just now, writing doc to ML cloud succeeded without the contentType.